### ISSUE TITLE
adapt-contrib-boxmenu: README out of date?

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2955]
In the [accessibility section of box menu's README](https://github.com/adaptlearning/adapt-contrib-boxMenu#accessibility), it states:
> Several menu-related elements are assigned a label using the [aria-label](https://github.com/adaptlearning/adapt_framework/wiki/Aria-Labels) attribute: **ariaRegion**, **menuItem**, and **menuEnd**. These labels are not visible elements. They are utilized by assistive technology such as screen readers. Should the label texts need to be customised, they can be found within the **globals** object in [*properties.schema*](https://github.com/adaptlearning/adapt-contrib-boxmenu/blob/master/properties.schema).

As far as I can see, these three labels are not actually used any more... if so I reckon this whole section can be deleted.

Equally, the [contentObjects.json section](https://github.com/adaptlearning/adapt-contrib-boxMenu#contentobjectsjson) mentions `durationLabel` but this is actually defined in course.json (as `_globals._menu._boxMenu.durationLabel`)
